build.gradle

```diff
@@ -1,19 +1,20 @@
 import com.linkedin.gradle.DistributeTask
 
 buildscript {
-    ext.kotlinVersion = '1.2.0'
+    ext.kotlinVersion = '1.4.30'
 
     repositories {
+        google()
```

Comment:
Could you elaborate why we need `google() repository`? It does not look like we are adding new dependencies.

Reply:
Good question. The dependencies here were so old...when this was "modern", AGP was published to JCenter and maven.google.com didn't exist 😄 

Updating AGP required adding this, because that's where AGP is published now.

Reply:
That's good to know!